aspnetcore/security/authentication/disable-cookie.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Disable automatic cookie authentication in ASP.NET Core
+author: John0King
+description: Disable automatic cookie authentication in ASP.NET Core
+ms.author: riande
+ms.date: 5/22/2019
+uid: security/authentication/disable-cookie
+---
+# Disable cookie authentication redirects in ASP.NET Core
+
+By [Kirk Larkin](https://github.com/serpent5), [John King](https://github.com/John0King), and [Rick Anderson](https://twitter.com/RickAndMSFT)
+
+The [`[Authorize]`](xref:Microsoft.AspNetCore.Authorization.AuthorizeAttribute) attribute specifies that access to a controller or action method is restricted to users who meet the authorization requirement. When the user:
+
+* Is not authenticated or doesn't have access to the controller or action method.
```

Comment:
Only un-authenticated users are redirect to the sign-in page. Authenticated but not-authorized users are redirected to the Forbid page.

Reply:
@Tratcher  with the default templates you're redirected to the sign-in page. Is that because the default templates set up the sign-in page as the Forbid page?

Reply:
No? You should end up here https://github.com/aspnet/AspNetCore/blob/master/src/Identity/UI/src/Areas/Identity/Pages/V4/Account/AccessDenied.cshtml

Reply:
Which template with which kind of auth enabled?

Reply:
And which version?